doc/classes/ARVROrigin.xml

```diff
@@ -35,6 +35,8 @@
 	</methods>
 	<members>
 		<member name="world_scale" type="float" setter="set_world_scale" getter="get_world_scale">
+			This scale allows you to adjust this to the unit system you use in your game. Most AR/VR platforms will assume a unit size of 1 unit in your game world = 1 meter in the real world.
```

Comment:
- Allows you to adjust the scale to your game's units.
- platforms will assume -> platforms assume
-  a unit size of 1 unit in your game world = 1 meter in the real world

maybe clearer as
...assume a scale of 1 game world unit = 1 meter in the real world.